packages/create-redwood-app/src/create-redwood-app.js

```diff
@@ -178,23 +178,54 @@ const tasks = new Listr(
     },
     {
       title: 'Installing Packages',
-      task: async (ctx, task) => {
-        task.output = `...installing packages...`
-        return execa('yarn install', {
-          shell: true,
-          cwd: `${targetDir}`,
-        }).catch(() => {
-          ctx.stop = true
-          task.title = `${task.title} (or not)`
-          throw new Error('Yarn not installed. Cannot proceed.')
-        })
+      task: () => {
+        return new Listr([
+          {
+            title: 'Engine Check: Node and Yarn Version Requirements',
+            task: async(ctx, task) => {
+              return execa(`check-node-version --node $(node -p "require('./package.json').engines.node")\
+                --yarn $(node -p "require('./package.json').engines.yarn")`, {
+                shell: true,
+                cwd: `${targetDir}`,
```

Comment:
The programatic API might be easier to use, but this works, so YMMV.

https://www.npmjs.com/package/check-node-version#api-usage